client/my-sites/domains/index.js

```diff
@@ -68,6 +75,24 @@ function getCommonHandlers( {
 	return handlers;
 }
 
+function registerSiteDomainPage( { path, controllers } ) {
+	page(
+		path,
+		domainManagementController.domainManagementSiteContext,
+		siteSelection,
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore
```

Comment:
Is there a need for these comment lines?

Reply:
Removed the comment here https://github.com/Automattic/wp-calypso/pull/97956/commits/6abb2012e446e777e11c87696d65ce7760294a75

Also fixed the menu selection issue here https://github.com/Automattic/wp-calypso/pull/97956/commits/1de29b0be5d575ed3b7ac24ef73b2503235748f2. Basically moved it under site's overview and have also changed the URLs